client/app/calendar/tasks/create/page.tsx

```diff
@@ -0,0 +1,43 @@
+'use client'
+
+import React from 'react'
+import { FormContainer, Loading } from '@/_components/layout'
+import { Button, Form, Heading } from '@/_components/ui'
+import { useGetHouseQuery } from '@/_hooks/api'
+import { AssigneeField, DateField, NoteField, TitleField } from '../components'
+import { useCreateTask } from '../hooks'
+import { TaskFormType } from '../types'
+
+export default function CreateTask() {
+  const { form, onSubmit: onCreateTask, isPending } = useCreateTask()
+
+  const { data: house, isLoading } = useGetHouseQuery()
+  const houseMembers = house?.houseMembers || []
+
+  if (isLoading) return <Loading />
+
+  return (
+    <FormContainer>
+      <Heading title='Create Task' />
+      <Form {...form}>
+        <form onSubmit={onCreateTask} className='mx-auto mt-8 w-full'>
+          <div className='space-y-5'>
+            <TitleField<TaskFormType> control={form.control} name='title' />
+            <DateField<TaskFormType> control={form.control} name='date' />
+            <AssigneeField<TaskFormType>
+              control={form.control}
+              name='assigneeId'
+              houseMembers={houseMembers}
+            />
+            <NoteField<TaskFormType> control={form.control} name='note' />
+          </div>
+          <div className='mt-10 text-center'>
+            <Button type='submit' isLoading={isPending} disabled={isPending}>
+              Create
+            </Button>
+          </div>
+        </form>
+      </Form>
```

Comment:
If you consider the integration test, it is better to create `CreateTaskForm` and put it here instead of each field gathered here! For this reason, I usually don't make a Field component (it is too small) unless it's complicated.

Reply:
I see your point, but I created this way because fields components can be used in other pages such as expense pages(in this case, I need to move the files though) .
And also if I were to create CreateTaskForm, I would have to pass all the props to hide FormContainer and Heading comonent.

Reply:
Yeah Date Firled for sure you need to move to `_components` directory later or now. And you just import it to `CreateTaskForm` component. 

>And also if I were to create CreateTaskForm, I would have to pass all the props to hide FormContainer and Heading component.

I think I don't get this meaning so let's hear about that tomorrow!